clients/client/src/main/java/org/projectnessie/client/api/MergeTransplantBuilder.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (C) 2020 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.projectnessie.client.api;
+
+import javax.validation.constraints.Pattern;
+import org.projectnessie.model.Validation;
+
+public interface MergeTransplantBuilder<R extends MergeTransplantBuilder<R>>
+    extends OnBranchBuilder<R> {
+  R fromRefName(
+      @Pattern(regexp = Validation.REF_NAME_REGEX, message = Validation.REF_NAME_MESSAGE)
+          String fromRefName);
+
+  R keepIndividualCommits(boolean keepIndividualCommits);
```

Comment:
By default squash is enabled with this during merge and transplant. If user wants to avoid squash they need to explicitly set this? 
I think other way around is better.

**But, If we squash commits we lose time travel support of those individual commits,** 
So, do we really need to support this feature ?